cms/static/js/views/utils/view_utils.js

```diff
@@ -173,6 +174,33 @@ define(["jquery", "underscore", "gettext", "js/views/feedback_notification", "js
             return false;
         };
 
+        keywordValidator = (function () {
+            var regexp = /%%+[^%]+%%/g;
+            var keywords = ['%%USER_ID%%', '%%USER_FULLNAME%%', '%%COURSE_DISPLAY_NAME%%', '%%COURSE_END_DATE%%'];
```

Comment:
This should be named more specifically to differentiate it from the
other arrays of keywords, e.g. `keywordsSupported`.

Prefixing each of the keyword arrays like `keywordsSupported`,
`keywordsInvalid`, and `keywordsFound` make the variables easier to grok
at a glance.

I'm a big of some form of Hungarian notation; Spolsky provides an
interesting read, _Making Wrong Code Look Wrong_ [1].

[1] http://www.joelonsoftware.com/articles/Wrong.html